packages/flutter_tools/lib/src/web/file_generators/js/flutter.js

```diff
@@ -0,0 +1,363 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+if (!_flutter) {
+  var _flutter = {};
+}
+_flutter.loader = null;
+
+(function () {
+  "use strict";
+
+  const baseUri = document.baseURI || "/";
```

Comment:
Ensure that `baseUri` ends in `/`. It can be anything the user wants, right?

Reply:
Well, we have checks to make sure that the value starts AND ends with `/`. But I agree with you, it's a good idea to not assume anything here.